pennylane/devices/null_qubit.py

```diff
@@ -494,7 +494,7 @@ def execute_and_compute_vjp(
     # TODO: Remove when PL supports pylint==3.3.6 (it is considered a useless-suppression) [sc-91362]
     # pylint: disable=unused-argument
     def eval_jaxpr(
-        self, jaxpr: "jax.extend.core.Jaxpr", consts: list, *args, execution_config=None
+        self, jaxpr: "jax.extend.core.Jaxpr", consts: list, *args, execution_config=None, shots=None
```

Comment:
```suggestion
        self, jaxpr: "jax.extend.core.Jaxpr", consts: list, *args, execution_config=None, shots=Shots(None)
```

And make sure to update the `zeros_like` function too.